public/stylesheets/register.css

```diff
@@ -0,0 +1,64 @@
+.container{
+
+  height: 100%;
+  width: 100%;
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  text-align: center;
+}
+.container a{
+  text-decoration: none;
+  color: black;
+}
+.header{
+  margin-top: 200px;
+}
+.main{
+  margin-top: 50px;
+}
+.linput input{
+  margin-top: 10px;
+  height: 40px;
+  width: 300px;
+  border-radius: 15px;
+  padding-left: 20px;
+  font-size: 15px;
+}
+.btn{
+  border: 5px solid black;
```

Comment:
some of these properties seem to be universal across the template - maybe extract them to separate CSS class, instead of duplicating it (e.g. make "bordered" class that has this border, colors etc.)

then use multiple classes in HTML like this: `<div class="someCssClass anotherCssClass thirdCssClass">`

Reply:
I saw that "forgot" class and "new-account" class had all properties the same, except "background-color", but when I tried to move these into separate class, then I didn't know which selector to use to select first and second button from newly created nwe class so I let them be. But I removed "forgot" class in register.css because it wasn't used.